src/patterns/durations/index.njk

```diff
@@ -46,6 +46,12 @@ Use the [correct errors pattern](/patterns/error-validation) and show the error
     patternlibExample({"path": "components/duration/examples/duration-address-error/index.njk"})
 }}
 
+If only one of the duration fields is causing a validation error, you can add error styling to just that single field by setting the additional `error` parameter.
+
+{{
+    patternlibExample({"path": "components/duration/examples/duration-error/index.njk"})
+}}
```

Comment:
The example above is incorrect now that you've made this change to highlight single fields.  So I would change that first example to an 'all fields are empty' example, with the message “Enter how long you have lived at this address” (if @LauraC119 agrees to my comment below)